release_notes.md

```diff
@@ -3,6 +3,7 @@
 <!-- Please add your release notes in the following format:
 - My change description (#PR)
 -->
+- Update Python Worker Version to [4.29.0](https://github.com/Azure/azure-functions-python-worker/releases/tag/4.29.0)
 - Update Python Worker Version to [4.28.1](https://github.com/Azure/azure-functions-python-worker/releases/tag/4.28.1)
```

Comment:
Can you please remove the prior updates? The release notes should reflect the released state (4.29.0)

Reply:
Updated!